### PR TITLE
modified the do_GET in request handler

### PR DESCRIPTION
**Pull request by trobinson1097**
ticket #2 

## Changes Made 
Changed the path of /animals in do_GET to /games
created two game dictionaries 

## How to test 
git fetch tr_request_modifications
git checkout tr_request_modifications

makes sure the path in do_GET is set to games 
there should be two game dictionaries

### CODE REVIEW

File: request_handler.py [hunk 1]
```diff
@@ -26,12 +26,12 @@ def do_GET(self):
...
 
         # It's an if..else statement
-        if self.path == "/animals":
+        if self.path == "/games":
             # In Python, this is a list of dictionaries
             # In JavaScript, you would call it an array of objects
             response = [
-                {"id": 1, "name": "Snickers", "species": "Dog"},
-                {"id": 2, "name": "Lenny", "species": "Cat"}
+                {"id": 1, "name": "Monopoly", " number_of_players": 4, "category_id": 1},
```
Comment (stevebrownlee) [inline request_handler.py:33]: You have an extra space at the beginning of number_of_players key that needs to be removed

**Review by stevebrownlee — CHANGES_REQUESTED** [review]
Please check my comment. One small change needed.

**Review by stevebrownlee — APPROVED** [review]
Looks good :shipit:

**Comment by trobinson1097** [comment]
thank you!!! @stevebrownlee

**Pull request merged**